blocks/resources/block.php

```diff
@@ -50,6 +51,29 @@ function render_callback( array $attributes ) : string {
 	$resources_query = new AirpressQuery( 'Resources', 0 );
 	$resources_query->addFilter( '{Publish Status of Resource} = "Published"' );
 
+	if ( class_exists( 'TRP_TRANSLATE_PRESS' ) ) {
+		$trp           = TRP_Translate_Press::get_trp_instance();
+		$url_converter = $trp->get_component( 'url_converter' );
+		$language_code = $url_converter->get_lang_from_url_string();
+	}
+	if ( $language_code ) {
```

Comment:
If not set would cause a notice error. Try this?
```suggestion
	if ( isset( $language_code ) ) {
```